dom.bs

```diff
@@ -3040,12 +3042,49 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
- <li><p>If <var>nodes</var> contains one <a for=/>node</a>, then set <var>node</var> to
- <var>nodes</var>[0].
+ <li><p>Let <var>isScriptElement</var> be true if <var>parent</var> is an {{HTMLScriptElement}}
+ <a for=/>node</a>; otherwise false.
+
+ <li><p>Let <var>newNodes</var> be « ».
+
+ <li>
+  <p><a for=list>For each</a> <var>value</var> of <var>nodes</var>:
+
+  <ol>
+   <li>
+    <p>If <var>value</var> is a {{Text}} <a for=/>node</a> and <var>isScriptElement</var>
+    is true:
+
+    <ol>
+     <li><p>Let <var>newValue</var> be a new {{Text}} <a for=/>node</a> whose
+     <a for=Node>node document</a> is <var>document</var> and <a for=CharacterData>data</a> is the
+     result of calling <a abstract-op>Get Trusted Type compliant string</a>, with {{TrustedScript}},
+     <var>document</var>'s <a>relevant global object</a>, <var>value</var>'s <a
+     for=CharacterData>data</a>, "<code>HTMLScriptElement text</code>", and "<code>script</code>".
+
+     <li><p><a for=list>Append</a> <var>newValue</var> to <var>newNodes</var>.
+    </ol>
+
+   <li>
+    <p>Otherwise, if <var>value</var> is a {{TrustedScript}}:
+
+    <ol>
+     <li><p>Let <var>newValue</var> be a new {{Text}} <a for=/>node</a> whose
+     <a for=CharacterData>data</a> is <var>value</var>'s <a for=TrustedScript>data</a>
```

Comment:
`<a for=TrustedScript>data</a>` definitely should be linkifying but it's not. Possibly bikeshed hasn't reindexed it yet?

Reply:
Likely, yeah.

Reply:
Seems to work locally now after a bikeshed update.